clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -30,9 +30,9 @@ private const val SCAN_GROUP = "scans"
 private const val PROJECT_GROUP = "projects"
 
 /**
- * This function verifies that a request to FossID server was successful
- * @param operation the label of the operation, for logging
- * @param withDataCheck if true, the payload will be checked for existence
+ * Verify that a request for the given [operation] was successful.
+ * [operation] is a free label describing the operation.
+ * If [withDataCheck] is true, also the payload data is checked, otherwise that check is skipped.
```

Comment:
Please just make this continuous text (as usual wrapped at column 120) instead of using one sentence per line.